tests/config/intern-api.js

```diff
@@ -0,0 +1,11 @@
+define({
```

Comment:
I'm not faimiliar with intern, but why is this file needed when you're adding `tests/apiUnit` to intern-node.js?